tests/nat/middleware/test_defense_middleware_output_verifier.py

```
@@ -0,0 +1,278 @@
+            action="partial_compliance"
+        )
+        middleware = OutputVerifierMiddleware(config, mock_builder)
+
+        # Mock LLM response
+        mock_llm = AsyncMock()
+        mock_response = MagicMock()
+        mock_response.content = '{"threat_detected": false, "confidence": 0.9}'
+        mock_llm.ainvoke = AsyncMock(return_value=mock_response)
+        middleware._llm = mock_llm
+
+        async def mock_next(value):
```

Ruff (0.14.8)

Unused function argument: `value`

(ARG001)